requirements.txt

```diff
@@ -1,3 +1,5 @@
 # Switch back to pypi package when ops is updated with `container.can_connect`
 # ops
 git+https://github.com/canonical/operator/#egg=ops
+lightkube<=0.8.0
+jinja2<3.1
```

Comment:
Is this dependency actually required? I think I was able to template just with the `lightkube` dependency.